frontend/talentsearch/src/js/components/Home/partials/Opportunities/Opportunities.tsx

```diff
@@ -94,7 +92,7 @@ const Opportunities = () => {
                   "Heading for the digital government job opportunities",
               })}
               link={{
-                href: diPaths.allPools(),
+                href: paths.allPools(),
```

Comment:
External links are busted up. :-(
![image](https://user-images.githubusercontent.com/8978655/202282528-84d32d34-ec0f-4d2d-a14b-5e96b00c347f.png)

Reply:
Good eye! 👓 

Fixed in [fix external links](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4597/commits/eec4412985348141a5598df987aff5d1a035e7a3)